src/Common/tests/Common.Tests.csproj

```diff
@@ -74,6 +74,7 @@
     <Compile Include="Tests\System\CharArrayHelpersTests.cs" />
     <Compile Include="Tests\System\IO\PathInternal.Tests.cs" />
     <Compile Include="Tests\System\IO\StringParserTests.cs" />
+    <Compile Include="Tests\System\BitOpsTests.cs" />
```

Comment:
Seems like you also need to include https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/System/BitOps.cs ?

Reply:
Perhaps this is not the right project to add these tests. I thought that methods in CoreCLR/CoreLib would be visible here after the sync-up?

Reply:
The file gets mirrored to the repo, but you'd still need to include it in this test project.

Reply:
Thanks. Fixed